12-ieee802154/test_spec12.py

```diff
@@ -0,0 +1,66 @@
+from time import sleep
+import pytest
+
+from riotctrl_shell.ieee802154 import IEEE802154Phy
+
+APP = 'tests/ieee802154_hal'
+PHY_MODE = 'O-QPSK'
+DEFAULT_CHANNEL = 26
+pytestmark = pytest.mark.rc_only()
+
+
+class Shell(IEEE802154Phy):
+    pass
+
+
+@pytest.mark.parametrize('nodes',
```

Comment:
```suggestion
@pytest.mark.local_only
@pytest.mark.parametrize('nodes',
```

`remote-revb` is not available on the testbed.